tensorflow_datasets/core/lazy_imports_lib.py

```diff
@@ -48,106 +52,106 @@ class LazyImporter(object):
 
   @utils.classproperty
   @classmethod
-  def apache_beam(cls):
+  def apache_beam(cls: Type[T]) -> object:
```

Comment:
There shouldn't be annotations for `cls` or `self`, as the annotations are implicit.
For the returned object, it should be Module type, instead of object.

Reply:
@Conchylicultor , Can you please give me a hint about how to return Module type using pytype annotations because I was not able to find it anywhere?

Reply:
From the first google page: https://www.google.com/search?q=typing+module+python

 https://stackoverflow.com/questions/48389157/python-typing-for-module-type

Reply:
done